sidebars.js

```diff
@@ -107,6 +107,14 @@ module.exports = {
         'reference/workflows/troubleshooting',
       ]
     },
+    { 
+      type: 'category',
+      label: 'Serverless Inference',
+      collapsed: false,
+      items: [
+        'reference/inferenceservice/getting_started',
```

Comment:
Let's use something else than `getting_started` so that this is confused with our main "Getting started" page. I suggest `overview` for the first page. For the other pages would be `create-with-web-ui`, `create-with-workflow-task` and `create-with-python-sdk`. Update page titles accordingly.